src/Apps/Order/Routes/Shipping2/index.tsx

```diff
@@ -455,9 +247,11 @@ export const ShippingFragmentContainer = createFragmentContainer(
   {
     order: graphql`
       fragment Shipping2_order on CommerceOrder {
+        ...FulfillmentDetailsForm_order
+        ...SaveAndContinueButton_order
```

Comment:
These two components can manage their own co-located data, vs needing to pass props